GVRf/Framework/framework/src/main/jni/vulkan/vulkanCore.cpp

```diff
@@ -1276,7 +1281,8 @@ void VulkanCore::InitPipelineForRenderData(const GVR_VK_Vertices* m_vertices, Vu
 
         // TODO: add shadowmap descriptor
 
-        vkShader->bindTextures(vkmtl, writes,  descriptorSet);
+         if(vkShader->bindTextures(vkmtl, writes,  descriptorSet) == -1)
```

Comment:
VulkanShader::bindTextures is confusing. It can't actually return -1. Please check it out; ``texIndex`` and ``fail`` are not being used.